components/UploadImage/UploadImage.js

```diff
@@ -0,0 +1,13 @@
+export default function UploadImage({ name, onChange, title }) {
+  return (
+    <>
+      <label htmlFor={name}>

```

Comment:
You might want to add an id to the input element that matches the htmlFor of the label to establish a clear association